### DISCUSSION TITLE
It is normal for it to look like this

### DISCUSSION THREAD

**Discussion by MrViPPer1** [lavalink-devs/Lavalink#801] [Q&A] [answered]
ERROR 1155 --- [  XNIO-1 task-2] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.31.128.1:56154
![image](https://user-images.githubusercontent.com/85508784/211172520-7f04ed38-325e-4c14-8caf-bfb34d6b9d9b.png)

**Comment by topi314** [comment, marked as answer]
normal in what sense? If you provide the wrong password yes

**Reply by MrViPPer1** [reply]
i put the right password

**Reply by topi314** [reply]
you clearly didn't, else it would work.
Make sure your application.yml is in the same folder as where you start lavalink from and it's actually valid

**Reply by MrViPPer1** [reply]
i put that password and i start by typing bash start.sh or by presing the start button
![image](https://user-images.githubusercontent.com/85508784/211174914-fe60f447-4c57-420b-a53a-7ce83a9ec4c2.png)

**Reply by MrViPPer1** [reply]
i dont know is a secure one or not

**Reply by topi314** [reply]
> i dont know is a secure one or not

I don't really understand what you mean by that, but just because you entered port `443` doesn't make it `https` or `wss`

your `application.yml` looks okay besides that

**Reply by MrViPPer1** [reply]
I do not know if what i need to put
![image](https://user-images.githubusercontent.com/85508784/211175416-7c798f99-2f8a-4e11-93f0-d842fe7b81c8.png)

**Reply by topi314** [reply]
prob false if you're not running https / wss

**Reply by topi314** [reply]
it seems like you're not even using official lavalink so this is the wrong place to ask for help in the first place
